appveyor.yml

```diff
@@ -38,10 +38,8 @@ install:
     - IF %PHP%==1 echo max_execution_time=1200 >> php.ini-min
     - IF %PHP%==1 echo date.timezone="UTC" >> php.ini-min
     - IF %PHP%==1 echo extension_dir=ext >> php.ini-min
-    - IF %PHP%==1 echo extension=php_openssl.dll >> php.ini-min
-    - IF %PHP%==1 echo extension=php_curl.dll >> php.ini-min
-    - IF %PHP%==1 echo curl.cainfo=c:\php\cacert.pem >> php.ini-min
     - IF %PHP%==1 copy /Y php.ini-min php.ini-max
+    - IF %PHP%==1 echo extension=php_openssl.dll >> php.ini-max
```

Comment:
yes sure, thx to https://github.com/symfony/symfony/pull/18276/files/82c6bed6b0f7f3f66ad0f98ae40c35dcad328575#diff-180360612c6b8c4ed830919bbb4dd459L53